src/components/Banner/README.md

```diff
@@ -500,6 +500,38 @@ class ModalExample extends React.Component {
 }
 ```
 
+### Banner with focus
+
+<!-- example-for: web -->
+
+Banner's can be focused through the ref API
```

Comment:
Use when a banner needs to be programmatically focused to help guide the user. For example, an error banner in a form after it's been submitted.

??

Reply:
@selenehinkley @sadiesaurus Do you both have any opinions?

Reply:
`Banner with focus receives focus programmatically when it appears. Use this component to communicate problems that merchants need to resolve before they can complete a task.` 

- In the component a11y documentation, we usually say "receive focus" instead of "be focused", so I'm favour of keeping things consistent. 
- The second line is adapted from the description for critical banners. I'm not sure if the same use case applies here, though. Does this work? 
- Should we mention when to use critical banners instead?

Reply:
Thanks @sadiesaurus I'll make the change 😄 I think for now we don't have to mention it's use with critical banners for now. But maybe it's something we could develop a guideline / pattern for?

Reply:
@sadiesaurus Since this won't only be used for critical banners I adjusted it slightly. What do you think, have any suggestions?

```
Banner can programmatically receive focus. Use this functionality to draw the merchants attention to the banner.
```

Reply:
Ah, cool, good point! A couple of questions for ya: 
- When you use this banner, do you get to choose whether it receives focus when it appears, or will it always receive focus when it appears? 
- If the former is true, then I think we can adjust the content a bit to make this clear
- If the latter is true, then do we need to explain when it's appropriate to redirect the merchant's attention?

Reply:
We'll get to choose when Banner receives focus it won't always be  when it appears! @sadiesaurus

Reply:
The key is "Banner can programmatically receive focus."

So unlike a prop which is set on the component and would automatically set focus the banner, this allows us to focus the banner at any given time. For instance when a user tries to submit the form.

Reply:
Cool, thank you for the context 🙏  

I think your adjustment works @AndrewMusgrave , with one wee punctuation edit: 
`the merchants attention` > `the merchant’s attention`